src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -213,13 +213,23 @@ case class MesosStatusUpdateEvent(
 
 case class InstanceChanged(
     id: Instance.Id,
+    runSpecVersion: Timestamp,
```

Comment:
2 options to go from here: make the internal state of Version OffsetDateTime or replace the Version altogether. We should make this consistent.